jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -585,6 +622,31 @@ YUI.add('user-profile', function() {
     },
 
     /**
+      Generate the details for the provided agreement.
+
+      @method _generateAgreementRow
+      @param {Object} agreement A agreement object.
+      @returns {Array} The markup for the row.
+    */
+    _generateAgreementRow: function(agreement) {
+      var term = agreement.term;
+      var createdAt = agreement.createdAt;
+      var date = createdAt.getUTCFullYear() + '-' +
+        ('0' + (createdAt.getUTCMonth() + 1)).slice(-2) + '-' +
+        ('0' + createdAt.getUTCDate()).slice(-2);
+      return (
+        <li className="user-profile__list-row twelve-col"
```

Comment:
I found that if it wasn't a twelve-col the element would collapse, so the borders ended up stacked at the top.